sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -278,6 +162,7 @@ module_platform_driver(skl_hda_audio)
 /* Module information */
 MODULE_DESCRIPTION("SKL/KBL/BXT/APL HDA Generic Machine driver");
 MODULE_AUTHOR("Rakesh Ughreja <rakesh.a.ughreja@intel.com>");
+MODULE_AUTHOR("Brent Lu <brent.lu@intel.com>");
```

Comment:
Probably mention this in the comment message if it is really needed?

Reply:
removed